test/common/network/dns_impl_test.cc

```diff
@@ -880,8 +880,7 @@ TEST_P(DnsImplAresFlagsForTcpTest, TcpLookupsEnabled) {
   server_->addCName("root.cnam.domain", "result.cname.domain");
   server_->addHosts("result.cname.domain", {"201.134.56.7"}, RecordType::A);
   std::list<Address::InstanceConstSharedPtr> address_list;
-  struct ares_options opts;
-  memset(&opts, 0, sizeof(opts));
+  struct ares_options opts {};
```

Comment:
while you're here, make this C++ style?
```suggestion
  ares_options opts{};
```

Reply:
done